### ISSUE TITLE
Proxied page items not showing

### ISSUE THREAD

**Issue by wayjake** [stephenou/fruitionsite#267]
This might be related to [this incident](https://github.com/stephenou/fruitionsite/issues/241#issuecomment-1707594854). I only pulled a partial version of one of the suggested fixes so it may be hard to communicate what my current setup looks like. 

But I wanted to open this ticket to see if anyone else is currently seeing any new issues with this library. 

Non working proxied site:
![image](https://github.com/stephenou/fruitionsite/assets/3623016/b410f5f4-998c-4920-8c8e-ed486731357d)

Working site directly:
![image](https://github.com/stephenou/fruitionsite/assets/3623016/6b958ada-386c-400d-8c99-9e3e43d77119)

non working embedded site:
![image](https://github.com/stephenou/fruitionsite/assets/3623016/bd2ddd08-3a7b-4a7f-8ed5-2c465f81df4c)

same site from the directly published link:
![image](https://github.com/stephenou/fruitionsite/assets/3623016/550db5e6-1674-4bbf-a514-218c6d29ef6e)

**Comment by wayjake** [comment]
It looks like adding a version ie) ?v=fb79c36fff3c4428827fe9eed59922b5 could be helping. 


  
  ```
if (slugs.indexOf(url.pathname.slice(1)) > -1) {
    const pageId = SLUG_TO_PAGE[url.pathname.slice(1)];
    return Response.redirect("https://" + MY_DOMAIN + "/" + pageId + "?v=fb79c36fff3c4428827fe9eed59922b5", 301);
} 
```

**Comment by wayjake** [comment]
yes, this fixed my issue. my code is somewhat deviated from the core code. I don't use the "MetaWriter" and I don't re-write URL's quite as hard as the default code wants. But, I'll leave this here in case this is a general change that affects more people. Thanks!

**Comment by is-cout** [comment]
after trying a lot of things in #241 this was what finally worked for me. Thank you for the fix!
